### PR TITLE
Feature/register page

### PR DESCRIPTION
**Pull request by Xaroz**
Currently working on the Register Page, only step 2 of the form is done

### CODE REVIEW

File: src/pages/RegisterPage/steps/Step1.tsx [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
-import React from 'react'
+import React, { useState } from 'react'
 import { css, jsx } from '@emotion/core'
 import BaseButton from '../../../components/button'
-import { IonContent, IonInput, IonPage } from '@ionic/react'
-import RegisterHeader from '../header/header'
+import { IonContent, IonInput, IonPage, IonRadioGroup } from '@ionic/react'
+import RegisterHeader from '../header/Header'
+import RadioCard from '../../../components/Card/RadioCard/RadioCard'
+import Order from '../../../assets/order.png'
```
Comment (Evalir) [inline src/pages/RegisterPage/steps/Step1.tsx:8]: Hmm, bit of a nitpick, but could we add the `Img` suffix to this? Right now if I took a glance at this, I'd think it's a component.

File: src/pages/RegisterPage/steps/Step1.tsx [hunk 2]
```diff
@@ -57,6 +70,66 @@ const Step1: React.FC<Step1Props> = ({
...
+                font-size: 14px;
+                font-weight: bold;
+                font-family: Maison Neue;
+                line-height: 19px;
+              }
+
+              /* ion-card {
+                margin-inline-start: 0px;
+                margin-inline-end: 0px;
+              } */
```
Comment (Evalir) [inline src/pages/RegisterPage/steps/Step1.tsx:94]: :)

**Review by Evalir — APPROVED** [review]
That was pretty good & quick to review! Just a few nitpicks and we can get this merged.

**Comment by Evalir** [comment]
@Xaroz @PseudoMera Is this ready to merge?

**Review by PseudoMera — APPROVED** [review]

**Comment by PseudoMera** [comment]
> @Xaroz @PseudoMera Is this ready to merge?

Yes

**Pull request merged**